spec/factories/item.rb

```diff
@@ -1,11 +1,11 @@
 FactoryBot.define do
   factory :random_item, class: Item do
-    sequence(:name)   {|n| "#{Faker::Commerce.product_name} #{n}"}
-    description       {Faker::Lorem.sentence}
-    price             {rand(1..100)}
-    sequence(:image)  {|n| "https://picsum.photos/id/#{1000 + n}/300/300"}
-    active?           {true}
-    inventory         {rand(1..100)}
-    association       :merchant, factory: :jomah_merchant
+    sequence(:name)   { Faker::Commerce.product_name }
+    description       { Faker::Hipster.sentence(word_count: 5) }
+    price             { Faker::Commerce.price }
+    sequence(:image)  { |n| "https://picsum.photos/id/#{1000 + n}/300/300" }
+    active?           { true }
+    inventory         { rand(1..100) }
+    merchant_id       { rand(1..100) }
```

Comment:
I'm finding that calling `create(:random_item)` will fail because it's trying to make an association with a merchant that doesn't exist (error - `Validation failed: Merchant must exist`). I might be calling it incorrectly, but you could try replacing line 9 with `association  :merchant, factory: :random_merchant`

Reply:
This is great! I didn't know this was possible.

Reply:
This is great! I didn't know this was possible.